fire/SFMainMod.F90

```diff
@@ -902,28 +911,55 @@ subroutine  crown_damage ( currentSite )
              currentCohort%fraction_crown_burned = 0.0_r8
+
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
-                ! Flames lower than bottom of canopy. 
-                ! c%hite is height of cohort
-                if (currentPatch%SH < (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft))) then 
-                   currentCohort%fraction_crown_burned = 0.0_r8
-                else
-                   ! Flames part of way up canopy. 
-                   ! Equation 17 in Thonicke et al. 2010. 
-                   ! flames over bottom of canopy but not over top.
-                   if ((currentCohort%hite > 0.0_r8).and.(currentPatch%SH >=  &
-                        (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)))) then 
-
-                           currentCohort%fraction_crown_burned =  (currentPatch%SH-currentCohort%hite*(1.0_r8- &
-                                EDPftvarcon_inst%crown(currentCohort%pft)))/(currentCohort%hite* &
-                                EDPftvarcon_inst%crown(currentCohort%pft)) 
-
-                   else 
-                      ! Flames over top of canopy. 
-                      currentCohort%fraction_crown_burned =  1.0_r8 
-                   endif
+                
+                ! height_cbb = clear branch bole height at base of crown (m)
+                ! inst%crown = crown_depth_frac (PFT)
+                crown_depth                          = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft) 
+                height_cbb                           = currentCohort%hite - crown_depth
+                passive_crown_FI                     = 0.0_r8  
+                ignite_crown                         = 0.0_r8  
+                currentCohort%passive_crown_fire_flg = 0
 
-                endif
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Note: crown_ignition_energy to be calculated based on foliar moisture content from FATES-Hydro
+                   ! EQ 3 Van Wagner 1977
+                   ! h = crown_ignite_energy (kJ/kg), m = foliar moisture content based on dry fuel (%)
+                   ! crown_ignite_energy = 460 + 26 * m
+
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   ! Van Wagner EQ4 FI = (Czh)**3/2 where z=crown base height,h=heat crown ignite energy, FI=fire intensity
+                   ! 0.01 = C from Van Wagner 1977 EQ4 for crown of base height 6m, 100% FMC, and FI 2500kW/m
+                   passive_crown_FI = (0.01_r8 * height_cbb *EDPftvarcon_inst%crown_ignite_energy(currentCohort%pft))**1.5_r8
+
+                   if (passive_crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
```

Comment:
@jkshuman , we (@pollybuotte @lmkueppers and myself) were looking more carefully at this if statement, and we convinced ourselves that it should change to:
```suggestion
                   if (currentPatch%FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
```
because as written it says that `height_cbb = 1` is less likely to pass the if test than a `height_cbb = 10` which is the opposite of what should happen. Similarly if you applied a moisture dependence (as you say in line 933), a smaller m would be less likely to pass the if test than a larger m, which again is the opposite of what should happen.

Reply:
Ah yes. Good catch!